### DISCUSSION TITLE
Who uses Stylus ?

### DISCUSSION THREAD

**Discussion by iChenLei** [stylus/stylus#2605] [Show and tell]
Hi, folks ! Do you know which open source project or company using Stylus as css preprocessor ? Welcome show Stylus user case !

1. open source project name / company name
2. project logo / company logo (in which case it might be used in the Stylus's readme / doc website etc)
3. project github link / comapany official website
4. why you use stylus and not sass/less ? (not required!)

Thanks !

**Comment by iChenLei** [comment]
> Case Example

1. vuepress
2. vuepress logo
<img src="https://vuepress.vuejs.org/hero.png" width="200px" />
3. project link

https://github.com/vuejs/vuepress

**Comment by zbryikt** [comment]
1. loading.io
2. logo
<img src="https://avatars.githubusercontent.com/u/28078873?s=400&u=b1f2793e621007f74ca4c433ef83e3546c879a32&v=4&s=100" width="200px" />
3.
https://github.com/loadingio

for example:
https://github.com/loadingio/loading.css
https://github.com/loadingio/css-spinner

**Comment by gidhon** [comment]
1. platframe

2. <p xmlns="http://www.w3.org/1999/html">
    <a href="https://platframe.com">
        <img src="https://platframe.com/assets/images/external/_linked/platframe.svg" width=100 height=100>
    </a>
</p>

3. https://github.com/platframe/platframe  
4. stylus was, and still is ahead of sass/less in a number of feature categories

**Comment by tomap** [comment]
1. hexo
2. 
![image](https://user-images.githubusercontent.com/153854/146687262-0a1ca29f-8850-4fa9-97b9-e30dc54597d9.png)

3. https://github.com/hexojs/hexo-renderer-stylus

**Comment by midvok** [comment]
1. Stylus (the browser extension to restyle web pages)
2.
![143300679-f012276d-4e19-4707-9a16-cba1b03fe617](https://user-images.githubusercontent.com/69433093/147816591-e0e836bc-ddbb-44f9-a09b-56e6f06bc3b4.png)
3. https://github.com/openstyles/stylus
4. Just trying Stylus and appreciate more scripting flexibility compared to Less (although more difficult to learn).
